app/classes/document_query.rb

```diff
@@ -25,19 +25,30 @@ class DocumentQuery
   DATE_AGGREGATION_FIELDS = %i[created
                                changed].freeze
 
-  attr_reader :language, :site_filters, :tags, :ignore_tags, :date_range,
-              :included_sites, :excluded_sites
-  attr_accessor :query, :search
+  attr_reader :audience,
+              :content_type,
+              :date_range,
+              :excluded_sites,
+              :ignore_tags,
+              :included_sites,
+              :language,
+              :mime_type,
+              :searchgov_custom1,
+              :searchgov_custom2,
+              :searchgov_custom3,
+              :site_filters,
+              :tags
+  attr_accessor :query,
+                :search
 
   def initialize(options)
     @options = options
-    @language = options[:language] || 'en'
-    @tags = options[:tags]
-    @ignore_tags = options[:ignore_tags]
     @date_range = { gte: @options[:min_timestamp], lt: @options[:max_timestamp] }
-    @search = Search.new
-    @included_sites = []
     @excluded_sites = []
+    @ignore_tags = options[:ignore_tags]
+    @included_sites = []
+    @search = Search.new
+    parse_filters(options)
```

Comment:
Minor point here: the options are already available to `#parse_filters` through `@options` (L45), so it's not necessary to pass in `options` as an argument.

Reply:
Thanks!  Updated.